ThreeBotPackages/threefold/tfgrid_workloads/actors/workload_manager.py

```diff
@@ -212,6 +213,8 @@ def _reservation_validate(self, reservation):
         if not reservation.data_reservation.expiration_reservation:
             raise j.exceptions.Value("expiration_reservation field is required")
 
+        self._validate_customer_signature(reservation)
```

Comment:
you cannot put this here. cause a reservation can be registered without any signature inside.